doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -13,7 +13,7 @@ community. This allows operators to have a common Kubernetes-based architecture
 that supports any conformant VNF or CNF deployed on it to operate as expected.
 The purpose of this chapter is to outline all the components required to provide
 Kubernetes in a consistent and reliable way. The specification of how to use
-these components is detailed in `Chapter 04 <chapter04.md>`__.
+these components is detailed in :ref:`chapter04:Chapter 04`.
```

Comment:
This does not render well: 
![image](https://user-images.githubusercontent.com/11458775/170250313-b67923d7-adf0-4bf5-8237-4acb0615a87d.png)

Reply:
apparently we have to use :doc: , not :ref:

Reply:
@rgstori within the document you can use "ref" -- here it would be :ref:`ref_arch/kubernetes/chapters/chapter04`
if you wanted to refer to a specific section you would follow it up with a ":<scetion name>"